pkg/executor/docker/executor.go

```diff
@@ -44,38 +45,31 @@ type Executor struct {
 	// used to allow multiple docker executors to run against the same docker server
 	ID string
 	// the storage providers we can implement for a job
-	StorageProvider storage.StorageProvider
```

Comment:
Docker executor no longer contains a storage provider and thus no longer needs to (or can) implement the `HasStorageLocally` and `GetVolumeSize` methods.